src/rpc/rawtransaction.cpp

```diff
@@ -192,17 +192,20 @@ UniValue getrawtransaction(const JSONRPCRequest& request)
 
             "\nExamples:\n"
             + HelpExampleCli("getrawtransaction", "\"mytxid\"")
-            + HelpExampleCli("getrawtransaction", "\"mytxid\" 1")
-            + HelpExampleRpc("getrawtransaction", "\"mytxid\", 1")
+            + HelpExampleCli("getrawtransaction", "\"mytxid\" true")
+            + HelpExampleRpc("getrawtransaction", "\"mytxid\", true")
         );
 
     LOCK(cs_main);
 
     uint256 hash = ParseHashV(request.params[0], "parameter 1");
 
+    // Accept either a bool (true) or a num (>=1) to indicate verbose output.
     bool fVerbose = false;
-    if (request.params.size() > 1)
-        fVerbose = (request.params[1].get_int() != 0);
+    if ((request.params.size() > 1) &&
+        ((request.params[1].isNum() && request.params[1].get_int() != 0) ||
+         (request.params[1].isTrue())))
```

Comment:
Note that isTrue doesn't *fail* if the value is not a bool, so this also accepts non-bool and non-int values. E.g.:
```
src/bitcoin-cli -regtest getrawtransaction 0000000000000000000000000000000000000000000000000000000000000000 '"b"'
error code: -5
error message:
No information available about transaction
```
Same for `{}` `[]` etc. Should probably raise an error in that case.